classpath/sslcontext.h

```diff
@@ -0,0 +1,14 @@
+#include <openssl/ssl.h>
+
+static SSL_CTX* create() {
+    SSL_load_error_strings();
+    SSL_library_init();
+    OpenSSL_add_all_algorithms();
```

Comment:
I would make a method call something like SSL_LOAD that is called in the static block in SSLContext.java that does this.  there is no reason to initialize this stuff in c when every SSLContext is made.